src/Resources/Wacc/Roles/Roles.ts

```diff
@@ -0,0 +1,13 @@
+import AbstractResource from '../../AbstractResource';
+
+export default class Roles extends AbstractResource {
+    initialise(): void {
+        this.resourcePath = '/roles';
+        this.defaultAccess = 'wacc';
+    }
+
+    public all: Function = this.createMethodFromPartialSpec({
```

Comment:
There's a newer way to do this method

```suggestion
    public all: (...args: HttpClientConfig) => HttpClientResponse = this.createMethodFromPartialSpec({
```